livingdocs-openapi.json

```diff
@@ -3906,6 +3906,57 @@
         }
       }
     },
+    "/api/v1/retresco/re-enrich": {
+      "post": {
+        "security": [
+          {
+            "bearerAuth": [
+              "retresco"
+            ]
+          }
+        ],
+        "tags": [
+          "Retresco re-enrich"
```

Comment:
Can we call the tag Webhooks or Integrations to be more generic for the future? And can you add the tag at the top of the document? Like this: https://github.com/livingdocsIO/openapi/pull/8/files#diff-d954713d1fb62f21db1f222da4ad1975223366cb07b230f6881eae5e69b7d250R16-R24